packages/next/src/build/utils.ts

```diff
@@ -70,6 +70,9 @@ import { AppRouteRouteModule } from '../server/future/route-modules/app-route/mo
 
 export type ROUTER_TYPE = 'pages' | 'app'
 
+// Use `print()` for expected console output
+const print = console.log
```

Comment:
What's the reason for moving this to a separate variable? Will the behavior change in a specific way?

Reply:
It's an unrelated refactor, was doing testing in that file then realize it's easy to mix with daily `console.log` and could be accidentally deleted so I renamed the original one